### ISSUE TITLE
Backlog no results - Manual snatches episodes

### ISSUE THREAD

**Issue by kgbasio** [SiCKRAGETV/sickrage-issues#1181]
##### Your operating system and python version
Windows 8 - 2.7.8 (default, Jun 30 2014, 16:08:48) [MSC v.1500 64 bit (AMD64)]

##### What you did
Added a show - waited - waited some more - forced backlog search - waited - manual search individual episodes

##### What happened
Added show with no issues. Episodes all set to 'wanted'. Nothing changed from the 'wanted' status unless I manually clicked the search button for each episode. When manual search button pressed episode is found and snatched.
 *I did notice that when I swap seach types __episode only to season only__ that it seems to find the opposite to what I want. The log below is from an episode only search. I've tried having fallback seach turned on. Doesn't make a difference.*

##### What you expected
Add show and episodes get snatched automatically

##### Log:
https://gist.github.com/1f3c4e47d5c834f38deb

**Comment by pct00** [comment]
+1

**Comment by abeloin** [comment]
Change in KAT and TPB search for episode
`2015-04-11 12:07:10 INFO     SEARCHQUEUE-BACKLOG-80379 :: [KickAssTorrents] :: Performing season pack search for The Big Bang Theory`

**Comment by pct00** [comment]
isn't season search only fallback?
i had it enabled and already configured for episode search only
i completelly deactivated the option but seems to be the same

**Comment by kgbasio** [comment]
Unless I have misunderstood - your saying to set it to episode only from season only.
From my original post:
"I did notice that when I swap seach types episode only to season only that it seems to find the opposite to what I want. The log below is from an __episode only__ search. I've tried having fallback seach turned on. Doesn't make a difference."

Either way, it seems to be randomly grabbing the episodes it feels like now... Seems to pick only ones with like zero or 1 seeder.
Have tried setting it to require keywords but that eliminates all results... Tedious.
